tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -439,6 +439,12 @@ void CorProfiler::RewritingPInvokeMaps(ComPtr<IUnknown> metadata_interfaces, Mod
         {
             native_profiler_file = GetCoreCLRProfilerPath();
         }
+
+        if (native_profiler_file.empty())
```

Comment:
What scenario does this fix? Do the current paths handle COM registration for the profiler?

Reply:
I added a fallback on 2b80926 for those scenarios where we can't extract the path, COM / GAC registration.

Reply:
Ohh so when we change the DLL name we'd also update the `const WSTRING native_dll_filename = WStr("DATADOG.TRACE.CLRPROFILER.NATIVE.DLL");` to match the new name, is that right?

Reply:
yup, I don't know if we can extract the filename from somewhere else.

Reply:
Makes sense, thanks for the explanations!